test/core/primitives/primitives_codec_test.cpp

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include <outcome/outcome.hpp>
+
+#include <primitives/block.hpp>
+#include <primitives/impl/scale_codec_impl.hpp>
+#include <scale/basic_stream.hpp>
+
+using namespace kagome::common;
+using namespace kagome::primitives;
+
+/**
+ * @class Primitives is a test fixture which contains useful data
+ */
+class Primitives : public testing::Test {
+ public:
+  Primitives()
```

Comment:
add //clang-format off /on, I don't think it is formatted with clang-format

Reply:
clang-format directives were removed, you probably are looking at diff between commits?